src/libtools/application.cc

```diff
@@ -59,25 +59,39 @@ void Application::ShowHelpText(std::ostream &out) {
 }
 
 void Application::ShowManPage(std::ostream &out) {
+  out << FormatManPage<globals::man>();
+}
+
+void Application::WriteManFile() {
+  ofstream man_file;
+  man_file.open(ProgramName() + "_man.rst");
```

Comment:
I would be nice if the output file would not be hardcoded.

My thinking is something along the line of add an `--helpout` option, that specifies the file and could be used by `--man`, `--tex` and `--rst`.

@JensWehner any opinions?

Reply:
Sure, in general I think that is sensible and makes it all more general. What would be a sensible default?

Reply:
If no output file name is given, just print it to the screen (stdout).

Reply:
In general printing to the screen might be enough as we can redirect all the output in CMake.

Reply:
Did that.